vignettes/publishing-extensions.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Publishing plumbertableau Extensions to RStudio Connect"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Publishing plumbertableau Extensions to RStudio Connect}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup, include = FALSE}
+library(plumbertableau)
+```
+
+[RStudio Connect](https://www.rstudio.com/products/connect/) is a publishing platform for sharing the work produced by data science teams. It supports content in both R and Python, including Shiny apps and Plumber APIs. You can publish content to RStudio Connect [in a number of ways](https://docs.rstudio.com/connect/user/publishing/), including 1-click publishing from the RStudio IDE.
+
+plumbertableau integrates seamlessly with RStudio Connect. You can host any number of plumbertableau extensions on RStudio Connect, and Connect will ensure that requests from Tableau are passed to the correct extension.
+
+## Setting up RStudio Connect {#connect}
+
+### Requirements {#requirements}
+
+- Tableau extensions are only supported in RStudio Connect [VERSION NUMBER] and above.
+- RStudio Connect must be configured with the following settings:
+  - [`TableauExtension.Enabled`](https://docs.rstudio.com/connect/admin/appendix/configuration/#TableauExtension) must be `True`.
```

Comment:
Should we call out that `true` is the default value for this setting?

Reply:
Oh, yes, we should.

Reply:
```suggestion
  - [`TableauExtension.Enabled`](https://docs.rstudio.com/connect/admin/appendix/configuration/#TableauExtension) must not be set to `False`.
```